packages/state-transition/src/cache/epochCache.ts

```diff
@@ -280,21 +281,28 @@ export class EpochCache {
     const currentActiveIndices: ValidatorIndex[] = [];
     const nextActiveIndices: ValidatorIndex[] = [];
 
+    const previousShufflingDecisionBlock = getShufflingDecisionBlock(state, previousEpoch);
+    const previousShufflingIn = opts?.shufflingGetter?.(previousEpoch, previousShufflingDecisionBlock);
+    const currentShufflingDecisionBlock = getShufflingDecisionBlock(state, currentEpoch);
+    const currentShufflingIn = opts?.shufflingGetter?.(currentEpoch, currentShufflingDecisionBlock);
+    const nextShufflingDecisionBlock = getShufflingDecisionBlock(state, nextEpoch);
+    const nextShufflingIn = opts?.shufflingGetter?.(nextEpoch, nextShufflingDecisionBlock);
```

Comment:
What does the `In` suffix mean? Can you comment above this block detailing the rationale and purpose of these lines?

Reply:
those are shufflings comeing from `ShufflingCache` provided by BeaconChain. Will refactor to `cachedNextShuffling` and add more comments 👍